packages/browser-wallet/src/popup/shared/DisabledAmountInput/DisabledAmountInput.tsx

```diff
@@ -0,0 +1,19 @@
+import clsx from 'clsx';
+import React from 'react';
+
+interface Props {
+    label: string;
+    note?: string;
+}
+
+/**
+ * Component that looks like the AmountInput, but displays the given label instead of providing the input field.
+ */
+export default function DisabledAmountInput({ label, note }: Props) {
+    return (
+        <label className="form-input">
+            <div className={clsx('disabled-amount-input__field', 'form-input__field')}>{label}</div>
+            {note && <div className="form-input__note">{note}</div>}
+        </label>
+    );
+}
```

Comment:
What? This seems weird to me.. why is it not possible to just disable the input in question instead?

Reply:
Yeah, sorry I guess I should have made a storybook :upside_down_face: 
But it is to make this effect:
![Screenshot from 2023-03-10 13-35-25](https://user-images.githubusercontent.com/87635671/224317537-acd51faf-8182-4dbb-a947-c55f3a1294dc.png)
![Screenshot from 2023-03-10 13-35-17](https://user-images.githubusercontent.com/87635671/224317541-e04ee5fc-77ae-41e8-bcac-5e269550fd94.png)

I guess you could have put it inside the input, but I thought it was cleaner to separate. :thinking:

Reply:
Also.. why are they a different size in the 2 screenshots linked?